data/human/human missions.txt

```diff
@@ -5416,3 +5416,345 @@ mission "Deal Change 2"
+			`	The ship suddenly lurches to the side, and you find yourself on the ground. "No need to panic, everything is under control", the bogus pilot exclaims.`
+			`	After a moment of relative calm, the world begins to spin around you with sickening speed. You hear alarms go off all over the ship, and sparks fly all around you. It seems that he managed to put the ship into a spin by overloading some of the engines.`
+			`	You try to get up, but it is impossible to find your balance in the wheeling ship. The alarms get more frantic with every passing second, until you hear a low rumbling, then a thundering explosion consumes everything.`
+				explode
+	on complete
+		conversation
+			`After landing, you open the doors to let your passengers out. You already see the middle-aged man approaching you.`
+			`	"Took you long enough! Anyway, I must get going now. Some people can't sit around all day."`
+			choice
+				`	(Ask him for payment.)`
+				`	(Let him go.)`
+					decline
+			`	"Pay you?! You would have crashed the ship, had I not helped you! I've practically flown your ship here, and you have the guts to ask for payment?"`
+			`	He turns his back to you and walks off. You consider calling spaceport security, but decide against it; the sooner you get rid of him, the better.`
+
+
+
+mission "Drunk Teen Hijack"
+	name "Group transport to <planet>"
+	description "Bring <bunks> teenagers to <destination>. They cannot afford to pay you."
+	passengers 5
+	source
+		attributes "dirt belt"
+	destination
+		distance 4
+		attributes "dirt belt"
+	to offer
+		random < 1
+		"flagship crew" > 5
+	on offer
+		conversation
+			`As you are wandering the spaceport, a group of teenage boys huddling in a corner catches your eye. They seem to be talking amongst themselves, sometimes throwing a glance towards you.`
+			choice
+				`	(Approach them.)`
+					goto ask
+				`	(Walk away.)`
+			`	Just as you turn away to leave, one of the teenagers straightens up and approaches you.`
+				goto introduction
+
+			label ask
+			`	"Can I help you?"`
+			`	One of the teens steps forward and introduces himself.`
+
+			label introduction
+			`	"Hello, I'm Howard. Me and my four friends are looking for transport off-world, to <destination>, but cannot afford to pay for it. Can you help us?"`
+			choice
+				`	"Sure."`
+					goto accept
+				`	"Why do you want to go there?"`
+				`	"Sorry, I'm not headed that way."`
+					goto decline
+			`	After a moment of silence, Howard replies. "We heard they have well-paying jobs over there, even for teenagers. And we don't want to work for the Syndicate or those minted folks on the paradise worlds."`
+			`	He pauses for a second, then continues, "We grew up in the dirt belt. This is our home. We don't want to leave all that behind." You hear something strange in his voice, but cannot quite make sense of it. It might just be their desperation.`
```

Comment:
```suggestion
			`	He pauses for a second, then continues, "We grew up in the Dirt Belt. This is our home, we don't want to leave all that behind." You hear something strange in his voice, but cannot quite make sense of it. It might just be their desperation.`
```

Reply:
```suggestion
			`	He pauses for a second, then continues, "We grew up in the Dirt Belt. This is our home. We don't want to leave all that behind." You hear something strange in his voice, but cannot quite make sense of it. It might just be their desperation.`
```